framework/Supports/Autoptimize.php

```diff
@@ -130,6 +129,50 @@ public function add_dns_prefetch( $content ) {
 		return $content;
 	}
 
+	/**
+	 * Add target and rel for links.
+	 *
+	 * @param string $content HTML content generated for the page.
+	 *
+	 * @return string
+	 */
+	public function add_target_rel( $content ) {
+		$tags_matches = [];
+
+		if ( preg_match_all( '#<a.*>.*<\/a>#Usmi', $content, $matches ) ) {
+			$tags_matches = array_merge( $tags_matches, $matches[0] );
+		}
+
+		if ( empty( $tags_matches ) ) {
+			return $content;
+		}
```

Comment:
not sure why we need 2 if() statements. and why we need $tags_matches.

we have only <a> tags, so this is only 1 if condition with preg_match_all()